crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -769,31 +783,33 @@ mod tests {
 		let temp_dir = TempDir::new()?;
 		let temp_dir_path = temp_dir.path();
 
-		// No directory path.
-		let file = temp_dir_path.join("chain-spec.json");
-		let result = prepare_output_path(&file)?;
-		// Expected path: chain-spec.json
-		assert_eq!(result, file);
-
-		// Existing directory Path.
-		for dir in ["existing_dir", "existing_dir/", "existing_dir_json"] {
-			let existing_dir = temp_dir_path.join(dir);
-			create_dir_all(&existing_dir)?;
-			let result = prepare_output_path(&existing_dir)?;
-			// Expected path: existing_dir/chain-spec.json
-			let expected_path = existing_dir.join(DEFAULT_SPEC_NAME);
-			assert_eq!(result, expected_path);
-		}
-
-		// Non-existing directory Path.
-		for dir in ["non_existing_dir", "non_existing_dir/", "non_existing_dir_json"] {
-			let non_existing_dir = temp_dir_path.join(dir);
-			let result = prepare_output_path(&non_existing_dir)?;
-			// Expected path: non_existing_dir/chain-spec.json
-			let expected_path = non_existing_dir.join(DEFAULT_SPEC_NAME);
+		let test_cases = [
+			("chain-spec.json", "chain-spec.json"),
+			("existing_dir", "existing_dir.json"),
+			("existing_dir/", "existing_dir/chain-spec.json"),
+			("non_existing_dir", "non_existing_dir.json"),
+			("non_existing_dir/", "non_existing_dir/chain-spec.json"),
+			("some_file	", "some_file.json"),
```

Comment:
This specific case fails when I run the test.
The fix is only to remove the white space: ("some_file", "some_file.json"),

Reply:
Sorry I might have added that by accident when changing tabs. Thanks!